ee/vellum_cli/push.py

```diff
@@ -30,8 +30,9 @@ def push_command(
     release_tags: Optional[List[str]] = None,
     dry_run: Optional[bool] = None,
     strict: Optional[bool] = None,
+    workspace: Optional[str] = None,
 ) -> None:
-    load_dotenv()
+    load_dotenv(dotenv_path=os.path.join(os.getcwd(), ".env"))
```

Comment:
needed this in order for it to look in the correct tmp directory during tests. don't love it